### PR TITLE
Change tests to use Transforms instead of Operations

### PR DESCRIPTION
**Pull request by kjohnson-hugo**

### CODE REVIEW

File: test/collaboration.test.ts [hunk 1]
```diff
@@ -1,111 +1,232 @@
...
       // Verify initial states.
+      expect(src.children).toEqual(input);
       expect(toSlateDoc(src.syncDoc)).toEqual(input);
       expect(toSlateDoc(dst.syncDoc)).toEqual(input);
-      expect(src.children).toEqual(input);
       expect(dst.children).toEqual(input);
 
-      // Apply test ops to src editor, propagate changes to dst editor.
-      await TestEditor.applySlateOpsToSlate(src, operations as Operation[]);
+      // Apply tranforms to src editor, propagate changes to dst editor.
```
Comment (chrisfsampaio) [inline test/collaboration.test.ts]:
```suggestion
      // Apply transforms to src editor, propagate changes to dst editor.
```

File: test/testEditor.ts [hunk 2]
```diff
@@ -102,18 +104,32 @@ export const TestEditor = {
...
-    await TestEditor.applySlateOpsToSlate(
-      e,
-      nodes.reverse().map((node) => {
-        return {
-          type: 'insert_node',
-          path: path,
-          node: node,
-        };
+  applyTransforms: async (e: TestEditor, transforms: TransformFunc[]) => {
+    await Promise.all(
```
Comment (chrisfsampaio) [inline test/testEditor.ts]: `Promise.all` doesn't execute its children serially, could that be a problem here?

File: test/collaboration.test.ts (hunk 1 above)
Comment (kjohnson-hugo) [reply test/collaboration.test.ts]: 👍

File: test/testEditor.ts (hunk 2 above)
Comment (kjohnson-hugo) [reply test/testEditor.ts]: Yeah, maybe so. And possibly for the other places that I'm using it.

Comment (kjohnson-hugo) [reply test/testEditor.ts]: Perhaps

```
  applyTransforms: async (e: TestEditor, transforms: TransformFunc[]) => {
    transforms.map(async (transform) => {
      await TestEditor.applyTransform(e, transform);
    });
  },
```

is the right answer?

**Review by chrisfsampaio — APPROVED** [review]

**Pull request merged**